core/platform/models.py

```diff
@@ -84,26 +98,51 @@ def import_models(cls, model_names):
 
     @classmethod
     def import_transaction_services(cls):
+        """Importing gae_transaction_services
+
+        Returns:
+            Module gae_transation_services from core.platform.transactions
+        """
         from core.platform.transactions import gae_transaction_services
         return gae_transaction_services
 
     @classmethod
     def import_current_user_services(cls):
+        """Importing the current user services
+
+        Returns:
+            Module gae_current_user_services from core.platform.users
+        """
         from core.platform.users import gae_current_user_services
         return gae_current_user_services
 
     @classmethod
     def import_app_identity_services(cls):
+        """Importing the required app identity services
+
+        Returns:
+            Module gae_identity_services from core.platform.app_identity
+        """
         from core.platform.app_identity import gae_app_identity_services
         return gae_app_identity_services
 
     @classmethod
     def import_email_services(cls):
+        """Importing the correct email services module according to
+        the feconf values
+
+        Returns:
+            Module gae_email_services from core.platform.email /
+            Module mailgun_email_services from core.platform.email
+
+        Raises:
+            Exception: Invalid email service
```

Comment:
`Exception: feconf.EMAIL_SERVICE_PROVIDER does not correspond to a valid email_services module.`